R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -1339,6 +1339,11 @@ test_that("column functions", {
   expect_equal(collect(select(df, bround(df$x, 0)))[[1]][2], 4)
 
   # Test to_json(), from_json()
+  arr <- list(listToStruct(list("name" = "bob")))
+  df <- as.DataFrame(list(listToStruct(list("people" = arr))))
+  j <- collect(select(df, alias(to_json(df$people), "json")))
+  expect_equal(j[order(j$json), ][1], "[{\"name\":\"bob\"}]")
+
```

Comment:
this might be a bit hard for regular users to create the DataFrame this way - is there a different way that still produces column that works with `to_json`?

that should probably be in the example in functions.R?